ibis/expr/window.py

```diff
@@ -204,7 +205,8 @@ def _validate_frame(self):
             )
 
         if self.max_lookback is not None:
-            if not isinstance(self.max_lookback, ir.IntervalValue):
+            if not isinstance(
+                    self.max_lookback, (ir.IntervalValue, pd.Timedelta)):
                 raise com.IbisInputError(
                     "'max_lookback' must be specified as an interval"
```

Comment:
Can you add `or pandas.Timedelta object` to the error message?

Reply:
Yes, added in next commit.